widgetry/src/widgets/mod.rs

```diff
@@ -351,6 +371,7 @@ impl Widget {
             false,
         )))
         .named(label)
+        // Why is this still required? The button Dropdown uses *already* has an outline
```

Comment:
This is an outstanding mystery to me, but it's a self contained mystery, so I thought it was time to move on to fixing something else.